bin/test

```diff
@@ -6,6 +6,9 @@ go get -t ./...
 
 rm -f coverage.txt
 
+unset AWS_ACCESS_KEY_ID
+unset AWS_SECRET_ACCESS_KEY
```

Comment:
I don't think there are any side effects (since we're executing `bin/test`, not sourcing it). The envvar isn't unset in the host environment permanently, only in the environment of the bash process that's running `bin/test`.

IMO in any test where an env var is accessed, we should set it explicitly, e.g. via:

- `Command: "FOO=bar convox install -h"`
- `os.Setenv(...)`
- in the `Env` field of the `ExecRun` struct)

...rather than relying on the environment of the user running the tests (with the obvious exceptions where we _want_ the envvar to affect the test suite, like `PKG` and `TEST`).